app/controllers/social_previews_controller.rb

```diff
@@ -36,6 +36,12 @@ def tag
     set_respond
   end
 
+  def comment
+    @comment = Comment.find(params[:id]) || not_found
```

Comment:
I just realized all the other actions follow this "find or not found" pattern but the `|| not_found` part is unnecessary because the find already raises not found in case the id is not valid

Reply:
Got this! Thanks for pointing that out, will update this. 😄